roles/postgres_replicant/tasks/main.yml

```diff
@@ -0,0 +1,45 @@
+# based on https://github.com/lesovsky/ansible-postgresql-sr-on-el6
+
+- name: stop postgresql on replicant
+  service:
+    name: postgresql
+    state: stopped
+
+- name: remove old postgresql cluster on replicant
```

Comment:
Thanks, that's helpful. I'm leaning toward belt-and-suspenders. Using that query plus a flag file (e.g. recovery.conf, maybe).